tests/integration/smb_share_test.go

```diff
@@ -273,6 +282,57 @@ func (s *SmbShareWithExternalNetSuite) TestServiceIsLoadBalancer() {
 	)
 }
 
+func (s *SmbShareSuite) TestMetricsOnPod() {
+	pod, cont, err := s.getMetricsContainer()
+	s.Require().NoError(err)
+	if cont == nil {
+		return // OK, running without metrics; ignored
```

Comment:
Please change this to a test skip. Example: `s.T().Skipf("no metrics container present")`

Reply:
ACK